frontend/src/components/profile/EditProfile.vue

```diff
@@ -58,8 +58,19 @@
                     />
                 </b-field>
 
-                <b-field label="Phone">
-                    <b-input v-model.trim="user.phone_number" />
+                <b-field
+                    label="Phone"
+                    :type="isPhoneInvalid ? 'is-danger' : ''"
+                    :message="isPhoneInvalid ?
```

Comment:
such notation is more readable
isPhoneInvalid 
 ?  'Please, enter a valid phone, e.g. +380950000000 or 380950000000':
 : ''